src/main/groovy/life/qbic/datamodel/accounting/Product.groovy

```diff
@@ -1,9 +1,11 @@
 package life.qbic.datamodel.accounting
 
+import life.qbic.datamodel.dtos.business.ProductCategory
+
 /**
- * Describes an entity which packages all {@link ProductItem}s of the same type
+ * Describes an entity which packages all {@link life.qbic.datamodel.dtos.business.ProductItem}s of the same type
  *
- * A product is of a defined by a {@link ProductCategory} and other traits that define how to handle items assigned to
+ * A product is of a defined by a {@link life.qbic.datamodel.dtos.business.ProductCategory} and other traits that define how to handle items assigned to
  * that product.
  *
  * @since: 1.0
```

Comment:
Are all the classes from `1.0` rather than `1.12.0` ?

Reply:
I didn't check when the original now deprecated product class was introduced into data-model-lib, but i can check if the versioning is correct tomorrow 👍